src/type/definition.ts

```diff
@@ -1196,8 +1200,9 @@ export class GraphQLArgument implements GraphQLSchemaElement {
     return {
       description: this.description,
       type: this.type,
-      defaultValue: this.defaultValue?.value,
-      defaultValueLiteral: this.defaultValue?.literal,
+      defaultValue: this.defaultValue,
+      externalDefaultValue: this.externalDefaultValue?.value,
```

Comment:
`externalDefaultValue` seems to be an overloaded term now.  I understand why we want to unify the value and literal in one property on the instance, but have them separate on the config, but having a properties with the same name on the instance and config, with different types seems more confusing.  

Would using something lie `instance.externalDefault?: { value: unknown } | { literal: ConstValueNode }` work here?  Not sure if `externalDefault` is too generic.

Reply:
I like `instance.externalDefault?: { value: unknown } | { literal: ConstValueNode }` or even `instance.default?: { value: unknown } | { literal: ConstValueNode }` as we are going to retire `defaultValue`

Reply:
tried your naming suggestion with the second variation above, added you as co-author, but let me know what you think!

Reply:
this looks great to me!